DFC.Digital/DFC.Digital.Web.Sitefinity.CourseModule.UnitTests/Views/CourseDetailsViewTests.cs

```diff
@@ -304,6 +305,45 @@ public void ShowOtherVenuesAndDatesTest(bool otherVenuesExists)
             }
         }
 
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void OtherVenuesStartdatesTest(bool startDateIsNull)
+        {
+            //Arrange
+            var startDate = DateTime.Now;
+
+            var courseDetailsCourseDetails = new _MVC_Views_CourseDetails_OtherDatesAndVenues_cshtml();
+            var courseDetailsViewModel = new CourseDetailsViewModel();
+            var courseDetails = new CourseDetails();
+
+            courseDetailsViewModel.CourseDetails.VenueDetails = new Venue();
+            courseDetailsViewModel.CourseDetails.Oppurtunities = new System.Collections.Generic.List<Oppurtunity>();
+            var otherDatesAndVenues = new Oppurtunity();
```

Comment:
Can we change to "Opportunity"? (...rather than "Oppurtunity")

Reply:
This is has been the name for it on the Sitefinity end since it was developed,  We have not changed it before as it will need to be done in quite a few files. I do not want to make code changes that are not related to this bug.